core/templates/components/state-editor/state-content-editor/state-content-editor.directive.html

```diff
@@ -49,7 +49,12 @@
     <span translate="I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE"></span>
   </div>
   <div class="mt-2">
+    <div ng-if="isCardContentLengthLimitReached()" class="oppia-card-height-limit-warning protractor-test-card-height-limit-warning mt-2">
+      <i class="material-icons oppia-hide-card-height-warning-icon protractor-test-hide-card-height-warning-icon" uib-tooltip="Hide warning" ng-click="hideCardHeightLimitWarning()">&#xe002;</i>
+      <span translate="I18N_CARD_CONTENT_LIMIT_WARNING_MESSAGE"></span>
+    </div>
```

Comment:
Do we have a red error CSS class instead of the yellow warning class you're using? I think that would make it more clear to the user!

Reply:
Done @EricZLou
![Screenshot from 2021-10-03 01-36-18](https://user-images.githubusercontent.com/44300735/135730780-f452fd1b-46f0-498f-a6da-7ada521742df.png)